mgmt_audio_decoder/src/sx_mgmt_audio_decoder.c

```diff
@@ -251,7 +253,9 @@ static UINT8 slice_start_find(
             sPES_EXT *pes_ext = (sPES_EXT *) curr_ptr;
 
             *pes_payload_size = ntohs(pes_ext->length) - 14;
-            assert(*pes_payload_size == 1920);
+
+            // PES packet length is zero for Sony Xperia
+            assert((*pes_payload_size == 1920) || (*pes_payload_size == -14));
```

Comment:
Isn't it better to do a dynamic check for Sony Xperia before the - 14 above?  As storing -14 isn't really valid as the payload size.  Could -14 being updated into the argument cause issues?